package.json

```diff
@@ -52,13 +52,13 @@
 			"title": "vscode-erg",
 			"properties": {
 				"vscode-erg.ergpath.path": {
-					"type": ["string" , "null"],
-					"default": null,
+					"type": "string",
+					"default": "",
 					"description": "Path to `.erg` directory"
 				},
-				"vscode-erg.executable.path": {
-					"type": ["string" , "null"],
-					"default": null,
+				"vscode-erg.executablePath": {
```

Comment:
I think it's common to use dots when classifying, so I thought it would be better not to separate these in this case